NitroxClient/GameLogic/Spawning/WorldEntities/PlaceholderGroupWorldEntitySpawner.cs

```diff
@@ -140,8 +106,8 @@ private IEnumerator SpawnWorldEntityChild(WorldEntity worldEntity, EntityCell ce
         {
             EntityMetadataProcessor.ApplyMetadata(worldEntityResult.value.Value, worldEntity.Metadata);
 
-            // For some reasons it's not zero after spawning so we reset it here
-            worldEntityResult.value.Value.transform.localPosition = Vector3.zero;
+            worldEntityResult.value.Value.transform.localPosition = worldEntity.Transform.LocalPosition.ToUnity();
+            worldEntityResult.value.Value.transform.localRotation = worldEntity.Transform.Rotation.ToUnity();
```

Comment:
```suggestion
            worldEntityResult.value.Value.transform.localRotation = worldEntity.Transform.LocalRotation.ToUnity();
```
Should be both local, right?